docs/metal.md

```diff
@@ -46,4 +46,5 @@ Command line interface for Equinix Metal
 * [metal ssh-key](metal_ssh-key.md)	 - SSH key operations: create, get, update, and delete.
 * [metal user](metal_user.md)	 - User operations: get and add.
 * [metal virtual-network](metal_virtual-network.md)	 - Virtual network (VLAN) operations : create, get, delete.
+* [metal vrf](metal_vrf.md)	 - VRF operations : create, TODO: make other commands.
```

Comment:
Can you add documentation of the new subcommand. Maybe we can have it as a draft PR till then